exporter/clickhouseexporter/config.go

```diff
@@ -137,14 +137,20 @@ func (cfg *Config) buildDB() (*sql.DB, error) {
 		return nil, err
 	}
 
-	// ClickHouse sql driver will read clickhouse settings from the DSN string.
-	// It also ensures defaults.
-	// See https://github.com/ClickHouse/clickhouse-go/blob/08b27884b899f587eb5c509769cd2bdf74a9e2a1/clickhouse_std.go#L189
-	conn, err := sql.Open(driverName, dsn)
+	opts, err := clickhouse.ParseDSN(dsn)
 	if err != nil {
 		return nil, err
 	}
 
+	if opts.ClientInfo.Products == nil {
+		opts.ClientInfo.Products = []struct {
+			Name    string
+			Version string
+		}{{"otel-exporter", getCollectorVersion()}}
```

Comment:
how about use the component name?
```suggestion
		}{{"otel-clickhouseexporter", getCollectorVersion()}}
```

Reply:
I thought about this, but consider that there's no other otel component that is exporting to ClickHouse, so we can exclude ClickHouse from name in this context. Also a shorter name would be preferred since it's simply less data.

I would even prefer `otel` or `otelcol`. Let me know what you think

Reply:
I prefer  `otelcol`

Reply:
`otelcol` it is. Thanks!